src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER = 2;
+    private static final int INITIAL_CAPACITY = 16;
+    private int currentCapacity = INITIAL_CAPACITY;
```

Comment:
Consider removing this redundant initializer for 'currentCapacity'; 'INITIAL_CAPACITY' is already defined as the initial capacity and you can use that constant directly.